docs/src/main/asciidoc/security-oauth2.adoc

```diff
@@ -16,7 +16,7 @@ It can be used to implement an application authentication mechanism based on tok
 
 This extension provides a light-weight support for using the opaque Bearer Tokens and validating them by calling an introspection endpoint.
 
-If the OAuth2 Authentication server provides JWT Bearer Tokens then you should consider using either xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication] or xref:security-jwt.adoc[SmallRye JWT] extensions instead.
+If the OAuth2 Authentication server provides JWT Bearer Tokens then you should consider using either xref:security-oidc-bearer-authentication-concept.adoc[OIDC Bearer authentication] or xref:security-using-jwt-tutorial[SmallRye JWT] extensions instead.
```

Comment:
Missing .adoc. Update to xref:security-using-jwt-tutorial.adoc[SmallRye JWT]

Reply:
Good catch. Fixing the fix in all the docs now.